inspirehep/modules/workflows/tasks/actions.py

```diff
@@ -827,3 +830,88 @@ def extract_authors_from_pdf(obj, eng):
                 grobid_authors_count
             )
     return obj
+
+
+def prepare_collaboration_multi_search(collaborations):
+    compiled_regexp = get_collaboration_cleaner_regexp()
+    multi_search = MultiSearch(index="records-experiments", using=current_search_client)
+    backup_multi_search = MultiSearch(index="records-experiments", using=current_search_client)
```

Comment:
do we really need the complexity of multi-searches here? there are a handful of collaborations at most, and we're not doing that in the matcher which sometimes matches hundreds of authors/references.

Reply:
IMHO multi search is not adding that much complexity, and TBH we should ease our ES a little as during harvests (when most WFS are running) we are getting the most of ES circuit breakers errors.